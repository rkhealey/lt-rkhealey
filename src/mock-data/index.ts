import profile from './profile.png';

export const user = {
  username: '@yourname',
  imageSrc: profile,
};

export const theme = {
  background_color: 'rebeccapurple',
  text_color: 'palegoldenrod',
};

export const links = [
  {
    url: 'http://www.google.com',
    text: 'Link One',
  },
  {
    url: 'http://www.google.com',
    text: 'Link Two',
  },
  {
    url: 'http://www.google.com',
    text: 'Link Three',
  },
  {
    url: 'http://www.google.com',
    text: 'Link Four',
  },
  {
    url: 'http://www.google.com',
    text: 'Link Five',
  },
];

export const showLinks = {
  title: 'Shows',
  links: [
    {
      date: 'April 01 2019',
      venue: 'The Forum, Melbourne',
      url: 'http://www.songkick.com/1',
      available: true,
    },
    {
      date: 'April 02 2019',
      venue: 'Venue Name, Canberra',
      url: 'http://www.songkick.com/2',
      available: false,
    },
    {
      date: 'April 03 2019',
      venue: 'Venue Name, Sydney',
      url: 'http://www.songkick.com/3',
      available: true,
    },
    {
      date: 'April 04 2019',
      venue: 'Venue Name, Brisbane',
      url: 'http://www.songkick.com/4',
      available: true,
    },
  ],
};
