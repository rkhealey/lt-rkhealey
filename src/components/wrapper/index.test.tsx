import React from 'react';
import { shallow } from 'enzyme';
import Wrapper from '.';

import { user } from '../../mock-data';

describe('Wrapper', () => {
  it('Renders', () => {
    const wrapper = shallow(
      <Wrapper {...user}>
        <p>Hello</p>
      </Wrapper>,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
