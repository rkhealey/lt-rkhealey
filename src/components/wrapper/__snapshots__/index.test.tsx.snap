// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wrapper Renders 1`] = `
<Styled(styled.div)
  variant="wrapper"
>
  <Avatar
    imageSrc="profile.png"
    username="@yourname"
  />
  <p>
    Hello
  </p>
</Styled(styled.div)>
`;
